src/AstromFit.cc

```diff
@@ -1094,7 +1094,7 @@ void AstromFit::makeMeasResTuple(const std::string &tupleName) const
     for (auto i = L.cbegin(); i != L.end() ; ++i)
```

Comment:
Turned most of those (in other files, too) into range-based loops, in the process making many more of them `const &`. Made other const loops `cbegin` to `cend` where I couldn't obviously make the loop range-based.